ec_grasp_planner/src/handarm_parameters.py

```diff
@@ -64,16 +65,12 @@ def __init__(self, **kwargs):
                                                                                     tra.rotation_matrix(
                                                                                         math.radians(180.), [1, 0, 0])))
 
-        # above the object, in hand palm frame
+        # position of hand relative to the object before and at grasping
         self['surface_grasp']['object']['pregrasp_transform'] = tra.concatenate_matrices(
-            tra.translation_matrix([-0.05, 0.0, 0.0]), tra.rotation_matrix(math.radians(10.0), [0, 1, 0]))
+            tra.translation_matrix([0.0, 0.0, 0.0]), tra.rotation_matrix(math.radians(10.0), [0, 1, 0]))
```

Comment:
did we tune also surface grasp params? this is the general object, @JannisW did you try surface grasp with this parameters?

Reply:
I tuned surface grasp parameter, before the wall grasps were tuned and I'm pretty sure that they have been improved right before the demo again.
I did surface grasps on a netbag, using this branch and bottom-3, yes.